lib/msf/ui/console/command_dispatcher/developer.rb

```diff
@@ -91,7 +91,7 @@ def modified_file_paths(print_errors: true)
     @modified_files |= files.map do |file|
       next if file.end_with?('_spec.rb') || file.end_with?("spec_helper.rb")
       File.join(Msf::Config.install_root, file)
-    end
+    end.compact
```

Comment:
Unrelated fix; Noticed that the reload functionality would crash on `nil` paths when changing spec files